.github/workflows/docs.yml

```diff
@@ -69,7 +69,9 @@ jobs:
           popd
 
       - name: Preview using surge
-        if: ${{ github.event_name == 'pull_request_target' && github.repository == 'alibaba/GraphAr' }}
+        # if: ${{ github.event_name == 'pull_request_target' && github.repository == 'alibaba/GraphAr' }}
+        # FIXME: Error: auth() received invalid user or password
+        if: false
```

Comment:
You could ignore the error in this PR, or, closing this one and open a new pull request then everything could work.

Reply:
Don't skip that in main branch.